stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -616,6 +616,19 @@ public class MessageListViewModel(
         chatDomain.deleteMessage(message, hard).enqueue()
     }
 
+    /**
+     * Removes the flag actions from our [messageActions], as well as the overlay, before flagging
+     * the selected message.
+     *
+     * @param message Message to delete.
+     */
+    public fun flagMessage(message: Message) {
+        messageActions = messageActions - messageActions.filterIsInstance<Flag>()
+        removeOverlay()
+
+        chatClient.flagMessage(message.id).enqueue()
```

Comment:
We need a visual feedback when something fails or succeeds, for example:
- When there was a failure deleting message with disabled offline support
- When there was a failure muting a user
- When there was a failure flagging a message
- Etc.

Reply:
Good point - I'll create tickets for this.